LICENSE.md

```diff
@@ -0,0 +1,141 @@
+# Apache License
```

Comment:
License files are pretty boilerplate. I wouldn't rename them or alter the content in any way, as it might impact how automated tools detect them.

Reply:
I had to fix this one, because it referenced the older "MIT" license, which we're not using any more.

Reply:
We seem to have a mixture of plain text and markdown formatted files for these LICENSE.md entries. I think I'll standardise on the markdown-formatted style.

Reply:
I might leave that revision for later.

Reply:
I reverted the change.